PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -849,13 +891,21 @@ bool AutoCompletion::setLanguage(LangType language) {
 				}
 			}
 		}
-
-		sort(_keyWordArray.begin(), _keyWordArray.end());
-
+		if (_ignoreCase)
+		{
+			std::sort(_keyWordArray.begin(), _keyWordArray.end(), [](const auto& lhs, const auto& rhs){

```

Comment:
This is an exact copy of the `std::sort()` call on line 120 and 376. You can probably put this into a separate function and would make the code a bit more readable, e.g. having a call to `sortCaseInsensitive(...)` or whatever.